### PR TITLE
Site redesign

### PR DESCRIPTION
**Pull request by samajammin**
## Description

Long-running branch for the website redesign.

Along with many design & content changes, this involves a rewrite of the codebase into Next.js.

Latest preview deploy: https://deploy-preview-127--ecosystem-support.netlify.app/

## Related Issue
None. See Notion doc for epic info: https://www.notion.so/efdn/ESP-website-redesign-4c4c05c812424a22b290ac8e7cb67ed6

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,79 +2,60 @@
   "name": "esp-website",
   "private": false,
   "description": "The website for esp.ethereum.foundation",
-  "version": "2.0.0",
-  "author": "Sam Richards <sam.richards@ethereum.org>",
+  "version": "3.0.0",
```
Comment (samajammin) [inline package.json:5]: 👍

File: package.json [hunk 2]
```diff
@@ -2,79 +2,60 @@
...
-    "format": "prettier --write \"**/*.{js,jsx,json,md}\"",
-    "start": "yarn develop",
-    "start:static": "gatsby build && gatsby serve",
-    "serve": "gatsby serve",
-    "test": "jest"
-  },
   "repository": {
     "type": "git",
-    "url": "https://github.com/ethereum/esp-website"
+    "url": "https://github.com/ethereum/esp-website/tree/esp-redesign"
```
Comment (samajammin) [inline package.json:56]:
```suggestion
    "url": "https://github.com/ethereum/esp-website"
```

File: package.json [hunk 3]
```diff
@@ -2,79 +2,60 @@
...
   },
   "bugs": {
-    "url": "https://github.com/ethereum/esp-website/issues"
+    "url": "https://github.com/ethereum/esp-website/tree/esp-redesign/issues"
```
Comment (samajammin) [inline package.json:59]:
```suggestion
    "url": "https://github.com/ethereum/esp-website/issues"
```

**Comment by samajammin** [comment]
Ready to ship 👍👍

File: package.json (hunk 2 above)
Comment (nhsz) [reply package.json:56]: updated!

File: package.json (hunk 3 above)
Comment (nhsz) [reply package.json]: updated!

**Review by pettinarip — APPROVED** [review]

**Pull request merged**